Ronni-ronsbons/week-3/hw giffaw-master/scripts/app.js

```diff
@@ -0,0 +1,80 @@
+$(document).ready(function() {
+  // Upon page load, show Top 25 gifs
+  let key = 'zbimCMBPhJzgKJkfTxlFf4tGndzeAzO2';
```

Comment:
I like that way you called your key here in your js, but a key was provided already in your HTML form. Due to you then calling with the api key twice you would receive error messages on your front end for dual requests. This is why when given a set of base code (which will happen alot in your job) it is a good practice to read through it and make sure you are not giving yourself extra work.